include/af/lapack.h

```diff
@@ -200,6 +200,26 @@ namespace af
     */
     AFAPI array inverse(const array &in, const matProp options = AF_MAT_NONE);
 
+#if AF_API_VERSION >= 37
+    /**
+       C++ Interface for pseudo-inverting (Moore-Penrose) a matrix.
+       Currently uses the SVD-based approach.
+
+       \param[in] in is the input matrix
+       \param[in] tol defines the lower threshold for singular values from SVD
+       \param[in] options determining various properties of matrix \p in
```

Comment:
Singular option.

Perhaps state that this must be AF_MAT_NONE

\param[in] option must be AF_MAT_NONE. For future use.

Reply:
Yup just changed it.